test/javascripts/adapaters/topic-list-test.js.es6

```diff
@@ -0,0 +1,11 @@
+module("adapter:topic-list");
+
+import { finderFor } from 'discourse/adapters/topic-list';
+
+test("finderFor", function() {
+  var mock = sandbox.mock(Discourse);
+  mock.expects("ajax").withArgs("/search.json?q=test%25%25");
```

Comment:
It would be better if you used pretender to mock the AJAX calls. I don't like using sinon mocks for this purpose.

You should be able to add this to `create-pretender` and have it intercepted automatically in test mode.